building/components/homeworld-admin-tools/src/configuration.py

```diff
@@ -173,6 +173,9 @@ def __init__(self, kv: dict):
                     command.fail("in config: multiple supervisors not yet supported")
                 self.keyserver = node
 
+    def is_kerberos_enabled(self):
```

Comment:
I'm a little confused about this computation.  What does the `root_admins` key usually mean?  Also, how exactly does the disabled-kerberos configuration work?

Reply:
The `root_admins` key refers to the list of kerberos principals that can authenticate as cluster root. If none are registered, we don't need kerberos auth.

Disabled kerberos configuration means no keygateway, no checking that the keygateway works, no keytab uploading, and needing to grant ssh certificates in a different way.

Reply:
How are ssh certs granted in this case?

Reply:
See https://github.com/sipb/homeworld/commit/91e081ecc9a3bdd742f7edabef5fc594e6f7daef

[moved here from other incorrectly-added email comment]